easybuild/easyconfigs/c/CHARMPP/CHARMPP-8.0.0-foss-2023a.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'ConfigureMake'
+
+name = 'CHARMPP'
```

Comment:
@robertosussex We strongly prefer sticking to the "official" software name, which in this case seems to be `Charm++`.